subworkflows/nf-core/fastq_align_dedup_bwamem/tests/gpu.nf.test

```diff
@@ -0,0 +1,224 @@
+nextflow_workflow {
+
+    name "Test Subworkflow FASTQ_ALIGN_DEDUP_BWAMEM"
+    script "../main.nf"
+    workflow "FASTQ_ALIGN_DEDUP_BWAMEM"
+    config "./nextflow.config"
+
+    tag "gpu"
+    tag "subworkflows"
+    tag "subworkflows_nfcore"
+    tag "subworkflows/fastq_align_dedup_bwamem"
+    tag "parabricks/fq2bam"
+    tag "samtools/index"
+    tag "picard/markduplicates"
+    tag "untar"
+
+    setup {
+        run("BWA_INDEX") {
+            script "../../../../modules/nf-core/bwa/index/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    file('https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa', checkIfExists: true)
+                ])
+                """
+            }
+        }
+
+        run("BWA_INDEX", alias: 'BWA_INDEX_PE') {
+            script "../../../../modules/nf-core/bwa/index/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    file('https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa', checkIfExists: true)
+                ])
+                """
+            }
+        }
+    }
+
+    test("Params: parabricks/fq2bam single-end | use_gpu ") {

```

Comment:
This doesn't follow nf-core [naming convetion guidelines ](https://nf-co.re/docs/guidelines/components/subworkflows#test-names). 

Please also update all other test names

```suggestion
    test("Sarscov2 fasta - SE - nodedup - GPU  ") {

```